avro-builder/builder/src/main/java/com/linkedin/avroutil1/builder/operations/codegen/own/AvroUtilCodeGenOp.java

```diff
@@ -172,7 +172,9 @@ public void run(OperationContext opContext) throws Exception {
       } catch (Exception e) {
         errorCount++;
         //TODO - error-out
-        LOGGER.error("failed to generate class for " + fullname + " defined in " + fileParseResult.getContext().getUri(), e);
+        System.err.println("failed to generate class for " + fullname + " defined in " + fileParseResult.getContext().getUri());
+        System.err.println(e.getMessage());
+        e.printStackTrace();
```

Comment:
this printStackTrace() is definitely an oops?

Reply:
This change was for local error printing. Updated PR